### ISSUE TITLE
commonlog and combined logs being generated with leading timestamp

### ISSUE THREAD

**Issue by danlsgiga** [caddyserver/caddy#2973]
Caddy v1.0.4

Log configuration:

```
(global-logs) {
  log / stdout "{combined} {hostonly} {request_id} {latency_ms}"  
  errors stderr {
    502 /local/error_pages/502.html
  }
}

site.local {
  import global-logs
  
  proxy / https://internal.local {
    transparent
  }
}
```

I tried setting the logging config as:
```
log / stdout "{remote} - {user} [{when}] \"{method} {uri} {proto}\" {status} {size} \"{>Referer}\" \"{>User-Agent}\" {hostonly} {request_id} {latency_ms}"
``` 

but that didn't help either.

My logs are being generated as (IP redacted):

```
2020/01/10 23:08:00 x.x.x.x - - [10/Jan/2020:23:08:00 +0000] "POST /api/oauth2/token HTTP/1.1" 200 3428 "-" "-" site.local ec9f8e64-8557-4db1-9583-3d14b0a590a1 122
```

when they were supposed to be:

```
x.x.x.x - - [10/Jan/2020:23:08:00 +0000] "POST /api/oauth2/token HTTP/1.1" 200 3428 "-" "-" site.local ec9f8e64-8557-4db1-9583-3d14b0a590a1 122
```

This is currently breaking our logstash grok parser because of the leading timestamp.

**Comment by mholt** [comment]
Oops, I think we need to revert #2781, which was apparently inadvertently a breaking change.

**Comment by danlsgiga** [comment]
I figured out a way of avoiding our grok parser not to fail, but this still needs a fix! :) Thanks sir.

**Comment by mholt** [comment]
@danlsgiga Thank you for your sponsorship! 💚  I'll try to drop back to the v1 branch and work on it this week.

**Comment by mholt** [comment]
@danlsgiga Are you able to build from source? Can you test the possible fix in #2976 when you have a chance?

**Comment by danlsgiga** [comment]
@mholt yup, I build from source! I can confirm your PR fixes the issue. Just pushed that build to my `dev` environment and I see logs being generated correctly now.

Thanks a lot!
